CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contribution guide
+
+Thank you for contributing to formalizations of the HoTT Book in rzk! In order to better organize the work it's better to follow these guidelines.
+
+## Code style
+
+[This](https://github.com/emilyriehl/yoneda/blob/master/STYLEGUIDE.md) style guide can be followed.
```

Comment:
I don't think we (will) follow that style guide entirely. In particular, we might want to keep the names used in the book as is, to stay closer to it and assist with readability (when reading formalisations and comparing against the book). But we can keep the reference to that style guide for now. Also, see #6.

Reply:
I will merge, despite the comment above.